File: oscexchange/src/main/java/ac/at/fhstp/digitech/oscexchange/errors/OSCPortClosingError.java
package ac.at.fhstp.digitech.oscexchange.errors;

import ac.at.fhstp.digitech.oscexchange.OSCPort;

public final class OSCPortClosingError extends OSCPortError {

    public OSCPortClosingError(OSCPort port) {
        super(port);
    }

}

File: oscexchange/src/main/java/ac/at/fhstp/digitech/oscexchange/errors/OSCPortClosingError.kt
package ac.at.fhstp.digitech.oscexchange.errors

import ac.at.fhstp.digitech.oscexchange.OSCPort

/**
 * An error that occurs when an open port cannot be closed
 */
class OSCPortClosingError(exception: Exception, port: OSCPort) :
    OSCPortError(exception, port)